Sources/Identity.swift

```diff
@@ -43,8 +44,16 @@ import Foundation
     }
 
     public func readyForEvent(_ event: Event) -> Bool {
-        return state.bootupIfReady(getSharedState: getSharedState(extensionName:event:),
-                                   createXDMSharedState: createXDMSharedState(data:event:))
+        guard state.bootupIfReady(getSharedState: getSharedState(extensionName:event:),
+                                  createXDMSharedState: createXDMSharedState(data:event:)) else {
+            return false
+        }
+
+        if event.urlVariables {
+            return getSharedState(extensionName: IdentityConstants.SharedState.Configuration.SHARED_OWNER_NAME, event: event, resolution: .lastSet)?.value != nil
```

Comment:
for this change we need to also bump the AEPCore dependency to 3.6.0 in AEPEdgeIdentity.podspec and Package.swift